app/upgrades/v0.1.14.go

```diff
@@ -0,0 +1,41 @@
+package upgrades
+
+import (
+	"context"
+
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	"github.com/pokt-network/poktroll/app/keepers"
+)
+
+const (
+	Upgrade_0_1_14_PlanName = "v0.1.14"
+)
+
+// Upgrade_0_1_14 handles the upgrade to release `v0.1.14`.
+// This upgrade has no state migrations, it only releases new onchain features:
+// - Add Morse supplier claiming non-custodial Morse owner check (#1317)
+var Upgrade_0_1_14 = Upgrade{
+	PlanName: Upgrade_0_1_14_PlanName,
+	// No KVStore migrations in this upgrade.
+	StoreUpgrades: storetypes.StoreUpgrades{},
+
+	// Upgrade Handler
+	CreateUpgradeHandler: func(
+		mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+		// Add new parameters by:
+		// 1. Inspecting the diff between v0.1.14-dev4..v0.1.14
```

Comment:
I would inspect the diff between `v0.1.13` and `v0.1.14` because `v0.1.13` is what is currently running onchain.